docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,900 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #195.                PL[i] = PL[i-1]
</pre><p> </p><p>If <code>pruned[i] == True</code> then the approximate value for <code>PL[i]</code> will be set to <code>PL[i-1]</code> . This results in having more than one global maxima. Although this is the approach of the paper, the MATLAB code shows a different approach:</p><p> </p><pre class="ql-syntax" spellcheck="false">% We subtract a very small number here to avoid the pruned
% subsequence having the same discord score as the real discord
Left_MP(i) = Left_MP(i-1)-0.00001;
</pre><p> </p><p>However, I think this is not an ideal approach as the value <code>0.00001</code> is hardcoded. Alternatively, we may do:</p><p> </p><pre class="ql-syntax" spellcheck="false"># see the end of the function _forward_process
mask = np.argwhere(D &lt; bsf) + chunk_start
pruned[mask] = True

# NEW
PL[mask] = D[D &lt; bsf] 
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<blockquote>However, I think this is not an ideal approach as the value&nbsp;<code>0.00001</code>&nbsp;is hardcoded.</blockquote><p> </p><p>I agree. This hack feels like it's a problem waiting to happen</p>

Reply:
<p>Could we please split up each function and place it in its own cell? This way, the discussion/comments can be more focused on each section of code. Is it worth submitting a new PR?</p>

Reply:
>  Is it worth submitting a new PR?

Now that you mentioned it, I think I may submit a new PR to clean things up.